.github/workflows/changelog.yml

```diff
@@ -0,0 +1,15 @@
+name: "Pull Request Changelog Workflow"
```

Comment:
Maybe "perf_changelog" would be more informative.

Reply:
Done in [2fb6ee8](https://github.com/lambdaclass/ethrex/pull/2120/commits/2fb6ee84d0f90d5904ce7d7cc8354487cbc4b794)